module/cluster_config.yaml.tmpl

```diff
@@ -0,0 +1,101 @@
+apiVersion: kops/v1alpha2
+kind: Cluster
+metadata:
+  name: {{.cluster_fqdn}}
+spec:
+  {{- if .kubeAPIServer }}
+  kubeAPIServer:
+    {{- range $key, $value := .kubeAPIServer }}
+    {{ $key }}: {{ $value }}
+    {{- end }}
+  {{- end }}
+  api:
+    loadBalancer:
+      type: Public
+  authorization:
+    RBAC: {}
+  channel: stable
+  cloudProvider: aws
+  configBase: s3://{{.kops_s3_bucket_id}}/{{.cluster_fqdn}}
+  etcdClusters:
+  - etcdMembers:
+  {{- range $index, $zone := splitList "," .zones }}
+    - instanceGroup: master-{{$zone}}
+      name: {{ substr ( int (sub (len $zone) 1 ) ) ( int ( len $zone) ) $zone }}
+  {{- end }}
+    name: main
+  - etcdMembers:
+   {{- range $index, $zone := splitList "," .zones }}
+    - instanceGroup: master-{{$zone}}
+      name: {{ substr ( int (sub (len $zone) 1 ) ) ( int ( len $zone) ) $zone }}
+  {{- end }}
+    name: events
+  iam:
+    allowContainerRegistry: true
+    legacy: false
+  kubernetesApiAccess:
+  - 0.0.0.0/0
+  kubernetesVersion: {{.kubernetes_version}}
+  masterPublicName: api.{{.cluster_fqdn}}
+  networkCIDR: 172.41.0.0/16
+  networkID: {{.vpc_id}}
+  networking:
+    {{.kubernetes_networking}}: {}
+  nonMasqueradeCIDR: 100.64.0.0/10
+  sshAccess:
+  - 0.0.0.0/0
+  topology:
+    dns:
+      type: {{.kops_dns_mode | title}}
+    masters: {{.kops_dns_mode}}
+    nodes: {{.kops_dns_mode}}
+  subnets:
+  {{- range $index, $zone := splitList "," .zones }}
+  - cidr: 172.41.{{mul 32 (add $index 1)}}.0/19
+    name: {{$zone}}
+    type: Public
+    zone: {{$zone}}
+  {{- end}}
+
+---
+
+{{- range $index, $zone := splitList "," .zones }}
+apiVersion: kops/v1alpha2
+kind: InstanceGroup
+metadata:
+  labels:
+    kops.k8s.io/cluster: {{$.cluster_fqdn}}
+  name:  master-{{$zone}}
+spec:
+  image: kope.io/{{$.ami_name}}
```

Comment:
What is this `kope.io`? Is it from the kops project?

Reply:
This one threw me off for the longest time, but yes, that's the URL they use.
https://github.com/kubernetes/kops/blob/2a5e13660d59184f6269c406a95ff49181408c8d/channels/stable#L19